authenticity-collector/src/services/collector-identity/collector-identity.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { IdentityClient, IdentityJson } from 'iota-is-sdk/lib';
+import * as fs from 'fs';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class CollectorIdentityService {
+	constructor(private configService: ConfigService, @Inject('IdentityClient') private identityClient: IdentityClient) {}
+
+	async checkCollectorIdentity() {
+		const id: string = this.configService.get('COLLECTOR_DID');
+		const secret: string = this.configService.get('COLLECTOR_SECRET');
+		let collectorIdentity: IdentityJson;
+		if (id && secret) {
+			try {
+				collectorIdentity = JSON.parse(fs.readFileSync('./collector-identity.json', 'utf-8'));
```

Comment:
Please remove all this section: you don't need to get file.
I guess this is old